aten/src/ATen/core/LegacyTypeDispatch.h

```diff
@@ -17,7 +17,7 @@ namespace at {
 //
 // NOTE [ Treating Variables as non-Variables in type dispatch ]
 //
-// What exactly does AutoNonVariableType do?  The short answer is, it causes
+// What exactly does AutoDispatchBelowAutograd do?  The short answer is, it causes
```

Comment:
We don't really use "below" as the name for this concept anywhere else, do we? Not that we should sweat this name too much, but we do use "after" in e.g. `DispatchKeySet::FullAfter`, so maybe `AutoDispatchAfterAutograd`?

Reply:
@bhosmer I named the other RAII guard `AutoDispatchBelowInplaceOrView` so we should either keep or change them both together I think ;)

Reply:
Ah you're absolutely right (and my bad for not raising the "below" thing there!) Anyway it's a very minor difference and the note explains exactly what the guard does in any case. Definitely not worth digging up the sidewalk.